.github/ISSUE_TEMPLATE/SIMPLE-TASK.yml

```diff
@@ -0,0 +1,41 @@
+name: "📋 Task"
+description: Create a new ticket for a task to go with a feature
+title: "📋 [TASK] - <title>"
```

Comment:
In general I recommend against unnecessary changes to more specific wording for a subcomponent because it makes the code less reusable.

However, if the interest of expediency, I have made the requested change.